src/test/java/Gumball/TaskListTest.java

```diff
@@ -0,0 +1,84 @@
+package Gumball;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
+
+public class TaskListTest {
+
+    @Test
+    public void add_simpleToDo_Success() throws InputErrorException {
```

Comment:
Test method names should follow the convention `unitBeingTested_descriptionOfTestInputs_expectedOutcome`, where of the 3 sections separated by underscore are written in camelCase. 

```suggestion
    public void add_simpleToDo_success() throws InputErrorException {
```
Same for the other methods in this file.